ui-components/src/page_components/listing/listing_sidebar/LeasingAgent.tsx

```diff
@@ -20,6 +20,8 @@ const LeasingAgent = (props: LeasingAgentProps) => {
     ? `tel:${listing.leasingAgentPhone.replace(/[-()]/g, "")}`
     : ""
 
+  const managementWebsite = listing.managementWebsite ? `http://${listing.managementWebsite}` : ""
```

Comment:
Can you make this guarded by a prop and then pass that prop in when we instantiate it in the `sites/public`?

Reply:
I put the whole "display management company details" behind the `showManagementCompany` prop - let me know if you were thinking something else?

Reply:
Yeah you're right - I tested what it would do if I took out `listing.managementCompany` and `listing.managementWebsite`, and the frontend builds and displays okay, but VSCode points out the error.

I added a `managementCompany` field to the prop.